src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -39,13 +56,28 @@ public FunctionsScaleManager(IScaleMonitorManager monitorManager, IScaleMetricsR
         /// <returns>The scale vote.</returns>
         public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusContext context)
         {
-            var monitors = _monitorManager.GetMonitors();
+            var scaleMonitors = _monitorManager.GetMonitors();
+            var targetScalers = _targetScalerManager.GetTargetScalers();
 
+            Utility.GetScaleInstancesToProcess(_environment, _functionsHostingConfiguration, scaleMonitors, targetScalers,
+                out List<IScaleMonitor> scaleMonitorsToProcess, out List<ITargetScaler> targetScalersToProcess);
+
+            var targetScalerVotes = await GetTargetScalersResult(context, targetScalersToProcess);
+
+            return new ScaleStatusResult
+            {
+                Vote = await GetScaleMonitorsResult(context, scaleMonitorsToProcess, targetScalerVotes.Select(x => x.Vote)),
+                TargetWorkerCount = targetScalerVotes.Any() ? targetScalerVotes.Select(x => x.TargetWorkerCount).Max() : null
```

Comment:
nit: simplify the max operation to targetScalerVotes.Max(p => p.TargetWorkerCount)